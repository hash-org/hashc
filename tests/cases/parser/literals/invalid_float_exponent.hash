// run=fail, stage=parse

k := 12e23.0;
