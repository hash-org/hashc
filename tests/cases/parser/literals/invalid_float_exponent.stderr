error: expected field name access or a method call
 --> $DIR/invalid_float_exponent.hash:3:12
2 |   
3 |   k := 12e23.0;
  |              ^ here
